R/fancy_alignment_plot.R

```diff
@@ -25,7 +25,7 @@
 #' @return A plot as a ggplot object (grob).
 #'
 #' @rawNamespace import(data.table, except = c("last", "first", "between", "transpose", "melt", "dcast"))
-#' @import ggplot2 cowplot dplyr
+#' @import ggplot2 GAMBLR.helpers dplyr
```

Comment:
One of the issues this PR addresses is dependency burden, so instead of adding this package I re-implemented this with a superior `pivot_longer` function.